controller/api/public/test_helper.go

```diff
@@ -344,7 +344,8 @@ func GenStatTsResponse(resName, resType string, resNs []string, basicStats bool,
 					Type:      resType,
 					Name:      resName,
 				},
-				TimeWindow: "1m",
+				TimeWindow:     "1m",
+				MeshedPodCount: 1,
```

Comment:
(2/2) This is the significant part of this PR. The other changes are test files updates that have gotten through their initial merges